src/commands/console/index.js

```diff
@@ -10,12 +10,184 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-const { Command } = require('@oclif/command')
+const aioConsoleLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-console', { provider: 'debug' })
+const config = require('@adobe/aio-cli-config')
+const { Command, flags } = require('@oclif/command')
+const { getToken, context } = require('@adobe/aio-lib-ims')
+const sdk = require('@adobe/aio-lib-console')
+const { CLI } = require('@adobe/aio-lib-ims/src/context')
+const Help = require('@oclif/plugin-help').default
+const yaml = require('js-yaml')
 
-class IndexCommand extends Command {
+const { CONSOLE_API_KEY = 'TransporterJaeger1' } = process.env
+const ORG_TYPE_ENTERPRISE = 'entp'
+const CONSOLE_CONFIG_KEY = '$console'
 
+class ConsoleCommand extends Command {
+  constructor (argv, config) {
+    super(argv, config)
+    this.apiKey = CONSOLE_API_KEY
+  }
+
+  async run () {
+    const help = new Help(this.config)
+    help.showHelp(['console', '--help'])
+  }
+
+  async initSdk () {
+    const currConfig = await context.getCli()
+    this.imsEnv = (currConfig && currConfig.env) || 'prod'
+    await context.setCli({ '$cli.bare-output': true }, false) // set this globally
+    aioConsoleLogger.debug('Retrieving Auth Token')
+    this.accessToken = await getToken(CLI)
+    this.consoleClient = await sdk.init(this.accessToken, this.apiKey, this.imsEnv)
+  }
+
+  /**
+   * Output JSON data
+   *
+   * @param {object} data JSON data to print
+   */
+  printJson (data) {
+    this.log(JSON.stringify(data))
+  }
+
+  /**
+   * Output YAML data
+   *
+   * @param {object} data YAML data to print
+   */
+  printYaml (data) {
+    // clean undefined values
+    data = JSON.parse(JSON.stringify(data))
+    this.log(yaml.safeDump(data, { noCompatMode: true }))
+  }
+
+  /**
+   * print current selected console config
+   *
+   * @param {object} [options] printOptions
+   * @param {string} [options.alternativeFormat] can be set to: 'json', 'yml'
+   */
+  printConsoleConfig (options = {}) {
+    const config = {}
+    config.org = this.getConfig('org.name')
+    config.project = this.getConfig('project.name')
+    config.workspace = this.getConfig('workspace.name')
+
+    // handling json output
+    if (options.alternativeFormat === 'json') {
+      this.printJson(config)
+      return
+    }
+    if (options.alternativeFormat === 'yml') {
+      this.printYaml(config)
+      return
+    }
+
+    // handling text output
+    if (!config.org) {
+      this.log('You have not selected any organization.\n You can use `aio org list` and `aio org select <org-name>` to select your org.')
+      return
+    }
+    this.log('You are currently in:')
+    this.log(`1. Org: ${config.org}`)
+    if (config.project) {
```

Comment:
aren't we printing out something if they don't have a project or workspace  as well? like `<not selected>` or something